src/contexts/ExchangeIssuance/types.ts

```diff
@@ -0,0 +1,24 @@
+export type IssuancePriceDisplay = {
+  from_quantity: string
+  from_token_price_usd: string
+  to_quantity: string
+  to_token_price_usd: string
+  input_value_usd: string
+  output_value_usd: string
+  gas_price_usd: string
+  gas_price_eth: string
+  slippage: string
+  total_price_usd: string
+  total_price_currency: string
+}
```

Comment:
Shouldn't be relevant since you're not using the tokensets api

Reply:
fixed